src/plugins/telemetry/telemetry_impl.h

```diff
@@ -381,5 +393,24 @@ class TelemetryImpl : public PluginImplBase {
     bool _has_received_gyro_calibration{false};
     bool _has_received_accel_calibration{false};
     bool _has_received_mag_calibration{false};
+
+    // Ardupilot calibration status values
+
+    struct ap_calibration_offset {
+        float x{0};
+        float y{0};
+        float z{0};
+
+        [[nodiscard]] bool calibrated() const { return ((x != 0) && (y != 0) && (z != 0)); }
```

Comment:
Fun, never seen `nodiscard` before

Reply:
Might not be useful, but since all my usages of this are nondiscarding I put it here.